src/openrct2/peep/peep.c

```diff
@@ -1392,12 +1392,14 @@ static uint8 peep_get_action_sprite_type(rct_peep* peep)
 	}
 }
 
-void sub_693B58(rct_peep* peep){
+void peep_update_current_action_sprite_type(rct_peep* peep){
```

Comment:
We don't need to write original addresses now. The git history is good enough should we ever need to know what they were.